pkgs/by-name/li/libguytools/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchurl
+, libsForQt5
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libguytools";
+  version = "2.1.0";
+
+  src = fetchurl {
+      url = "mirror://sourceforge/project/libguytools/libguytools/LatestSource/tools-${finalAttrs.version}.tar.gz";
+      hash = "sha256-eVYvjo2wKW2g9/9hL9nbQa1FRWDMMqMHok0V/adPHVY=";
+  };
+
+  qmakeFlags = [
+    "trunk.pro"
+    "toolsstatic.pro"
+  ];
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ libsForQt5.qmake libsForQt5.wrapQtAppsHook ];
+  dontWrapQtApps = true;
+  buildInputs = [ libsForQt5.qtbase ];
+
+  postPatch = ''
+    sed -i "/dpkg-buildflags/d" tools.pro
+    patchShebangs create_version_file.sh
+  '';
+
+  preConfigure = ''
+    ./create_version_file.sh
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r lib $out/
+    cp -r include $out/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A small programming toolbox";
+    mainProgram = "libguytools";
+    homepage = "https://libguytools.sourceforge.io";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = platforms.unix;
+    license = licenses.gpl2Plus;
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ d3vil0p3r ];
    platforms = lib.platforms.unix;
    license = lib.licenses.gpl2Plus;
```

Reply:
Also, is this supported on darwin? Project page only mentions linux support, and CI fails to build it on darwin